datasets/us/trade_csl/crawler.py

```diff
@@ -8,10 +8,83 @@
 from zavod import Context, Entity
 from zavod import helpers as h
 
+#     Programs that we emit in this crawler (is_ofac=False):
+#     "Nonproliferation Sanctions (ISN) - State Department",
+#     "ITAR Debarred (DTC) - State Department",
+#     "Unverified List (UVL) - Bureau of Industry and Security",
+#     "Entity List (EL) - Bureau of Industry and Security",
+#     "Denied Persons List (DPL) - Bureau of Industry and Security",
+#     "Military End User (MEU) List - Bureau of Industry and Security",
+
+#     Programs that we primarily emit in OFAC (is_ofac=True):
+#     "Capta List (CAP) - Treasury Department",
+#     "Palestinian Legislative Council List (PLC) - Treasury Department",
+#     "Foreign Sanctions Evaders (FSE) - Treasury Department",
+#     "Non-SDN Chinese Military-Industrial Complex Companies List (CMIC) - Treasury Department",
+#     "Non-SDN Menu-Based Sanctions List (NS-MBS List) - Treasury Department",
+#     "Specially Designated Nationals (SDN) - Treasury Department",
+#     "Sectoral Sanctions Identifications List (SSI) - Treasury Department",
 
 REGEX_AUTHORITY_ID_SEP = re.compile(r"(\d+ F\.?R\.?)")
 
 
+def emit_relationship(
+    context: Context,
+    entity_id: str,
+    *,
+    related_name: str,
+    list_entry: Dict[str, Any],
+    source_program: str
+):
+    related_entity = context.make("LegalEntity")
+    related_entity.id = context.make_id(related_name)
+    related_entity.add("name", related_name)
+    context.emit(related_entity)
+
+    make_and_emit_sanction(
```

Comment:
Do we need to emit a sanction here? If I understand this correctly, this is mostly "Huawei Inc. Affiliated Entity: Special Lab Ltd.". This sanction means that we not only sanction Special Lab Ltd (which happens in the main function) but also "Huawei Inc." Is this what we want, conceptually? Don't they have their own "you are sanctioned" list entry?

Reply:
I'm a bit worried that after we merge these into Huawei Inc., we'll have 12 Sanctions objects on Huawei that we then need to dedupe.

Reply:
Acutally, looking at https://www.opensanctions.org/statements/NK-P3xHZasb3VZasvJLDwiGHC/?prop=name, it seems like they (always?) have the parent company with the same name. 

I guess we could build a dict of name → entity ID where we then look up the parent... but that seems a bit overkill, we can let the dedupe process take care of it I guess.